packages/components/navbar/src/NavbarItem/NavbarItem.styles.ts

```diff
@@ -44,6 +44,7 @@ export const getNavbarItemStyles = ({ title }) => ({
       boxSizing: 'border-box',
       transition: `color ${tokens.transitionDurationShort} ${tokens.transitionEasingCubicBezier}`,
       borderRadius: tokens.borderRadiusMedium,
+      height: '30px',
```

Comment:
Is it the fix to avoid the loading skeleton & complete state jump?

Reply:
yes, but i had a chat with Andi and we have found a different approach, will be fixed with a different pr, but for time beeing, this is necessary so the buttons with icon only and a tooltip don't become too small